scripts/generate_known_standard_library.py

```diff
@@ -61,10 +37,9 @@ class FakeApp:
             "sre",
```

Comment:
These hardcoded ones should probably be removed and change to `modules = {}` -- we haven't had `sre` since 2.7.  I suspect these were added to work around bugs in the sphinx inventory?

Reply:
https://github.com/astral-sh/ruff/pull/11409